forc-plugins/forc-doc/src/doc.rs

```diff
@@ -21,23 +21,31 @@ pub(crate) struct Document {
 }
 impl Document {
     /// Creates an HTML file name from the [Document].
-    pub(crate) fn html_file_name(&self) -> String {
+    pub(crate) fn html_filename(&self) -> String {
```

Comment:
Fwiw, the Rust std uses an underscore separator, e.g. https://doc.rust-lang.org/std/path/struct.Path.html#method.file_name, but I don't really mind either way as long as we're consistent :+1:

Reply:
Gotcha, as I was working through this it just bothered my OCD brain for some reason haha